src/rhsm/connection.py

```diff
@@ -187,10 +187,9 @@ class ConnectionType(enum.Enum):
 
 # FIXME: this concept with reslib_class is horrible. The _request() method can return anything.
 # It does not have much benefit and it is easy to introduce new bug.
```

Comment:
You removed the restlib_class argument. :+1: Thus this FIXME comment should be removed too.